spec/spec_helper.rb

```diff
@@ -37,12 +37,23 @@ module Homebrew
   module Utils
     module_function
 
-    def popen_read(_cmd)
-      ""
+    def popen_read(cmd)
+      if cmd == "/bin/launchctl list | grep homebrew"
+        "77513   0       homebrew.mxcl.php"
```

Comment:
This (and other conditionals) feel a bit weird to add. Would be preferable to use stubs/mocks in the relevant test(s) instead.

Reply:
This is still a thing.